server/games/game.py

```diff
@@ -900,6 +902,7 @@ def to_dict(self):
             "num_players": len(connected_players),
             "max_players": self.max_players,
             "launched_at": self.launched_at,
+            "hosted_at": self.hosted_at,
```

Comment:
Need to convert to ISO string

Reply:
is timezone information required ?

Reply:
Yes I think so. There should be a helper function in`server.timing.datetime_now` that takes care of the time zone.